main.js

```diff
@@ -1,25 +1,32 @@
 var level = require("level");
 var db  = level("./student-db", { valueEncoding: 'json'});
-
+var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probationary'];
 
 const prompt = require('prompt-sync')({sigint: true});
 
  var stId = prompt('Enter ID: ');
  var fullName = prompt('Enter full Name: ');
- var stdAge = prompt('enter age: ');
+ var stdAge = prompt('Enter age: ');
  var stdAddress = prompt('Enter address: ');
 
 acceptStudents(stId, fullName,stdAge, stdAddress);
 
-function acceptStudents(stId, fullName,stdAge){
-    db.put('student', {id: Number(stId) ,name: fullName, age: Number(stdAge), address: stdAddress, status: 'applying' }, function(err){  
+async function acceptStudents(stId, fullName,stdAge){
+    await db.put('student', {id: Number(stId) ,name: fullName, age: Number(stdAge), address: stdAddress}, function(err){  
         output();
+        scheduleInterview(stId, 'February 26, 2021');
     })
-
 }
```

Comment:
acceptStudents is not part of this issue. You should fix it on it's issue.

Its not good to mix the acceptStudents and scheduleInterview 
you used await on db.put but then passed callback, you should use either of the approach.